spec/models/manageiq/providers/inventory/persister/finders_spec.rb

```diff
@@ -37,24 +37,24 @@
   it "raises an exception passing bad primary index used by finder" do
     expected_error = "Finder has missing keys for index :manager_ref, missing indexes are: [:hardware]"
     expect do
-      persister.networks.lazy_find(:hardwares => "something", :description => "public")
+      persister.networks.lazy_find({:hardwares => "something", :description => "public"})
```

Comment:
@agrare This should no longer be necessary now for the inventory_refresh, gem, right?

Reply:
yeah, they're still needed 🤷 See the most recent commit.  I believe this PR makes them required to differentiate arguments and kwargs: https://github.com/ManageIQ/inventory_refresh/pull/112 cc @agrare

Reply:
That's not released yet it is only on master

Reply:
yay 🎉 ... I'll ride master and see if I can drop these changes